packages/timeline-state-resolver/src/integrations/quantel/connection.ts

```diff
@@ -591,11 +591,10 @@ export class QuantelManager extends EventEmitter {
 		})
 	}
 	public clearAllWaitWithPort(portId: string) {
-		if (!this._waitWithPorts[portId]) {
```

Comment:
There is a bug task already for this SOFIE-3419, as this is not a recent mistake and has always been there

Reply:
I did not see any use of `clearAllWaitWithPort` but it seemed clearly wrong, so I just changed it now.

Reply:
Oh right yeah, this quantel one isn't called anymore due to a mistake in the refactoring. so I guess this can be in whatever state you prefer as it is currently dead code. It is also refactored in r52 too, so expect a minor merge conflict there